include/language/en_us.lang.php

```diff
@@ -2400,6 +2403,8 @@
     'LBL_DELETE_BUTTON_KEY' => 'd',
     'LBL_DELETE_BUTTON_LABEL' => 'Delete',
     'LBL_DELETE_BUTTON_TITLE' => 'Delete',
+    'LBL_BULK_ACTION_BUTTON_LABEL' => 'BULK ACTION',
```

Comment:
Line 1985 has a typo:
   'LNK_DELETE' => 'Eelete',

Should be
   'LNK_DELETE' => 'Delete',